instrumentation/druid-0.2/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.alibaba")
+    module.set("druid")
+    versions.set("[0.2.6,)")
+    skip("1.0.30")
```

Comment:
```suggestion
    skip("1.0.30")
    assertInverse.set(true)
```

Reply:
I think your best option is to set the version range back to `versions.set("[0.2.6,)")` so you could add `assertInverse.set(true)`. The same approach is used in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/bc5398ce3fc3a07af3b1ebdaee1cc7802026ba68/instrumentation/logback/logback-appender-1.0/javaagent/build.gradle.kts#L9 Alternatively if there is a class that appears in 1.0 but does not appear in earlier versions you could use something like https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/bc5398ce3fc3a07af3b1ebdaee1cc7802026ba68/instrumentation/jetty/jetty-8.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/v8_0/Jetty8InstrumentationModule.java#L26-L28 to limit the range where the instrumentation works.

Reply:
@laurit Early versions may not have the DruidDataSourceStatManager.addDataSource method, or the parameters may be different, such as 
[0.2](https://github.com/alibaba/druid/blob/0.2/src/main/java/com/alibaba/druid/stat/DruidDataSourceStatManager.java#L59)
[0.2.5](https://github.com/alibaba/druid/blob/0.2.5/src/main/java/com/alibaba/druid/stat/DruidDataSourceStatManager.java#L74)

Reply:
@AlchemyDing sure, I think you can also use `versions.set("(,)")` if muzzle passes on all published versions.